### DISCUSSION TITLE
评论导入成功，但刷新后没有新的评论

### DISCUSSION THREAD

**Discussion by XiaoPika** [walinejs/waline#1232] [Q&A] [answered]
近期腾讯云和 Vercel 的政策均有变动，因此迁移一下评论到 Waline。

从 CloudBase 导出 Twikoo 的 json，粘贴到[数据迁移助手](https://waline.js.org/migration/tool.html)，然后生成 output.json，在 Waline 的后台选择导入，提示导入成功，回到评论管理界面，仍为空白，期间控制台和alert未见报错信息。

部署环境：阿里云函数+LeanCloud国内版

![image](https://user-images.githubusercontent.com/10910848/181172293-a2c75117-d0d9-441f-81b2-72df0b826f91.png)

能否请大佬指点一二？

**Comment by lizheming** [comment, marked as answer]
后台的导入导出暂时不支持其他的系统的数据。如果是其它的系统的迁移，LeanCloud 的话需要走 LeanCloud 官网的导入，导入之前你在迁移助手那拿到的数据

**Reply by XiaoPika** [reply]
谢谢！！
